docs/sources/helm-charts/tempo-distributed/get-started-helm-charts/_index.md

```diff
@@ -237,11 +314,11 @@ traces:
     enabled: false
 ```
 
-
 ### Optional: Add custom configurations
 
 You can use a YAML file, like `custom.yaml`, to store custom configuration options that override the defaults present in the Helm chart.
 The [tempo-distributed Helm chart's README](https://github.com/grafana/helm-charts/blob/main/charts/tempo-distributed/README.md) contains a list of available options.
+Any values that are enabled in `values.yaml` are on by default.
```

Comment:
I'm not happy with this wording. I added the sentence because I wanted users to understand that the values.yaml file provides the defaults for the helm installation. Oh. I should just state that. 
```suggestion
The `values.yaml` files provides the defaults for the helm chart. 
```